models/demos/falcon7b/tt/falcon_model.py

```diff
@@ -109,9 +148,18 @@ def model_preprocessing(self, llm_mode, input_ids, kv_cache_len, num_input_token
             assert sequence_size % 32 == 0, "For prefill, sequence_size must be multiple of 32!"
             assert kv_cache_len == 0, "For prefill, no kv_cache is passed in!"
 
-            tt_input_ids, attention_masks = [], []
-            # Create attn masks on host
+            tt_input_ids, tt_attention_mask = [], []
             for i, device in enumerate(self.devices):
+                tt_input_ids.append(
+                    ttnn.as_tensor(
+                        input_ids[i : i + 1],
+                        dtype=self.model_config["INPUT_DTYPE"],
+                        layout=ttnn.ROW_MAJOR_LAYOUT,
+                        device=device,
+                        memory_config=self.model_config["INPUT_MEMCFG"],
+                    )
+                )
+
```

Comment:
The changes to this function seem to mistakenly be reverting some recent changes [539750a]((https://github.com/tenstorrent/tt-metal/commit/539750a91d6545a17df69f26dd1185d4611be2f6)) for async optimizations